packages/runtime/src/widgets/Flex.tsx

```diff
@@ -0,0 +1,49 @@
+import {
+  useRegisterBindings,
+  type EnsembleWidget,
+} from "@ensembleui/react-framework";
+import { WidgetRegistry } from "../registry";
+import {
+  type EnsembleWidgetStyles,
+  type EnsembleWidgetProps,
+  type HasItemTemplate,
+} from "../shared/types";
+import { Column } from "./Column";
+import { Row } from "./Row";
+
+interface FlexStyles extends EnsembleWidgetStyles {
+  direction?: "horizontal" | "vertical" | undefined;
+  gap?: number;
+}
+
+export type FlexProps = {
+  children?: EnsembleWidget[];
+} & HasItemTemplate &
+  EnsembleWidgetProps<FlexStyles>;
+
+export const FlexWidget: React.FC<FlexProps> = (props) => {
+  const { "item-template": itemTemplate, children, ...rest } = props;
+  const { values } = useRegisterBindings({ ...rest });
```

Comment:
You don't need to call this since this is all handled by the underlying implementation